relayer/src/services/relayService/handleRelayRequest.ts

```diff
@@ -131,7 +115,8 @@ export async function handleRelayRequest(req: any, res: any) {
       `/handleRelayRequest - req.body.instructions: ${req.body.instructions}`,
     );
 
-    const job = await addRelayJob({ instructions });
+    const nonce = createNonce();
+    const job = await addRelayJob({ instructions, nonce });
```

Comment:
What's the purpose of the nonce?
I think it is not a good identifier for the job. The identifier should be based on the instructions, so that in case the connection to the user drops the user can query the state of it's tx.

Reply:
good point, replaced with stringified instructions' data concatenated  for job.id.
is used to ID jobs in queue (should be unique)

Reply:
what about hashing the stringified instructions data?
(now you have a job id which is really long/the actual data)